test/tools/ossfuzz/protoToAbiV2.cpp

```diff
@@ -204,12 +204,24 @@ std::string ProtoConverter::addressValueAsString(unsigned _counter)
 		.render();
 }
 
-std::string ProtoConverter::fixedByteValueAsString(unsigned _width, unsigned _counter)
+/// Returns a hex literal if _isHexLiteral is true, a string literal otherwise.
+/// The flag _encloseInDoubleQuotes is true by default. It is false only when
```

Comment:
Bug: If `_isHexLiteral` is false it returns a string that contains `_width*2` bytes. It should return `_width` bytes.